src/neo/Oracle/OracleRequest.cs

```diff
@@ -0,0 +1,50 @@
+using Neo.Cryptography;
+
+namespace Neo.Oracle
+{
+    public abstract class OracleRequest
+    {
+        /// <summary>
+        /// Only for hashing entropy
+        /// </summary>
+        public enum RequestType : byte
+        {
+            HTTP = 0x01,
```

Comment:
HTTPS?

Reply:
The request is the same, do you think that we need to difference in two objects?

Reply:
If we allow for both HTTP and HTTPS, it can be `HTTP`. If we allow HTTPS only, then it should be `HTTPS`.